command.sh

```diff
@@ -1,2 +1,3 @@
 cat licence | wc -l
 cat licence | grep -c CONNECTION
+rm *.*
```

Comment:
fuck!!! change this!!!

Reply:
please merge. I'm sure, I'm a best developer than you. No doubt @FabGlo